backend/de.metas.cucumber/src/test/resources/de/metas/cucumber/features/contract/callorder/callOrderContract.feature

```diff
@@ -48,7 +48,12 @@ Feature: Call order contract
 
     When the order identified by order_1 is completed
 
-    Then validate that order_1 is not invoiceable nor shippable
+    Then after not more than 30s, no M_ShipmentSchedules are found:
```

Comment:
AFAIU here we stall the whole build for another 30 + 30 seconds and it's already taking very long. hence my suggestion to invoke the handlers

Reply:
I've found the handlers in question, pls see stepDefs `there are no M_ShipmentSchedule for C_Order`, same for `C_Invoice_Candidate`